src/main/target/REVOLT/target.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "RVLT"
+#define USBD_PRODUCT_STRING     "Revolt"
+#define TARGET_DEFAULT_MIXER    MIXER_QUADX_1234
+
+
+
+#define LED0_PIN                PB5  
+
+#define USE_BEEPER
+#define BEEPER_PIN              PB4
+
+#define ENABLE_DSHOT_DMAR       true
+
+#define USE_TRANSPONDER
+
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+#define SERIALRX_PROVIDER       SERIALRX_SBUS
+#define SERIALRX_UART           SERIAL_PORT_USART1
+
+#define USE_TARGET_CONFIG
+
+#define USE_SERIAL_4WAY_BLHELI_INTERFACE
+
+
+#define INVERTER_PIN_UART1      PC0
+
+/*----------Spi Config--------*/
+#define USE_SPI
+
+#define USE_SPI_DEVICE_1
+#define SPI1_NSS_PIN            PA4
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_NSS_PIN            PB12
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+#define USE_SPI_DEVICE_3
+#define SPI3_NSS_PIN            PB3
+#define SPI3_SCK_PIN            PC10
+#define SPI3_MISO_PIN           PC11
+#define SPI3_MOSI_PIN           PC12
+
+/*----------I2C Config--------*/
+#define USE_I2C
+#define USE_I2C_DEVICE_1
+#define I2C_DEVICE              (I2CDEV_1)
+#define I2C1_SCL                PB8
+#define I2C1_SDA                PB9
+
+/*----------Gyro Config--------*/
+#define MPU6000_CS_PIN          PA4
+#define MPU6000_SPI_INSTANCE    SPI1
+
+#define MPU6500_CS_PIN          PA4
+#define MPU6500_SPI_INSTANCE    SPI1
+
+#define USE_GYRO
+#define USE_ACC
+
+#define USE_GYRO_MPU6500
+#define USE_GYRO_SPI_MPU6500
+#define GYRO_MPU6500_ALIGN      CW0_DEG
+
+#define USE_ACC_MPU6500
+#define USE_ACC_SPI_MPU6500
+#define ACC_MPU6500_ALIGN       CW0_DEG
+
+
+#define USE_EXTI
+#define MPU_INT_EXTI            PC4
+#define USE_MPU_DATA_READY_SIGNAL
+
+/*----------Flash Config--------*/
+#define M25P16_CS_PIN           PB3
+
+#if defined(REVOLTOSD)
+#define M25P16_SPI_INSTANCE     SPI2
+#else
+#define M25P16_SPI_INSTANCE     SPI3
+#endif
+
+
+#define USE_FLASHFS
+#define USE_FLASH_M25P16
+
+/*----------OSD Config--------*/
```

Comment:
Since we have two targets for now, we can as well put  `#if defined(REVOLTOSD)` around the OSD driver configuration.

Reply:
Changes made, let me know if you need anything else